README.md

```diff
@@ -1,3 +1,46 @@
 # kecho
 
 This is a lightweight echo server implementation in Linux kernel mode.
+
+## Usage
+
+Run make to build kecho and the benchmarking tool:
+```
+$ make
+```
+
+After the module is loaded, you can use `telnet` to interact with the module:
+```
+$ telnet 127.0.0.1 12345
+```
+
+If the default port (`12345`) is in use or you simply want to use another port, you can specify it when loading the module. Also, `backlog` is specifiable, which has default value `128`:
+```
+$ sudo insmod fastecho.ko port={port_you_want} backlog={amount_you_want}
+```
+
+Note that too much concurrent connection would be treat as sort of DDOS attack (mainly caused by configs (kernel: `tcp_max_syn_backlog` and `somaxconn`. Application (fastecho): `backlog`). Nevertheless, default maximum number of fd per-process is `1024`. These limitation can cause performance degration when benchmarking, if you insist to proceed the benchmark, perform following modifications: (PROCEDURE MAY VARY BETWEEN DIFFERENT DISTROS)
+
+### Ubuntu
```

Comment:
It is not Ubuntu specific. In fact, it is common for GNU/Linux.

Reply:
Oh, thanks for correcting me!